android/measure/src/main/java/sh/measure/android/config/MeasureConfig.kt

```diff
@@ -188,6 +189,16 @@ class MeasureConfig(
      * through the app.
      */
     override val trackFragmentLoadTime: Boolean = DefaultConfig.TRACK_FRAGMENT_LOAD_TIME,
+
+    /**
+     *  Map of HTTP headers to be included in http request.
+     *  Defaults to an empty map. The following headers are always excluded:
+     *  * "Content-Type"
+     *  * "msr-req-id"
+     *  * "Authorization"
+     *  * "Content-Length"
+     */
```

Comment:
The documentation says this is a map for HTTP headers, while it accepts an interface. Can be updated to:

```kotlin
    /**
     * Allows configuring custom HTTP headers for requests made by the Measure SDK to the
     * Measure API. This is useful only for self-hosted clients who may require additional
     * headers for requests in their infrastructure.
     *
     * See [MsrRequestHeadersProvider] for usage details.
     */
```